src/main/scala/zio/sqs/producer/Producer.scala

```diff
@@ -205,42 +195,34 @@ object Producer {
    * @tparam T type of the event to publish.
    * @return result of the operation.
    */
-  private[sqs] def runSendMessageBatchRequest[R, T](client: SqsAsyncClient, failedQueue: Queue[SqsRequestEntry[T]], retryDelay: Duration, retryMaxCount: Int)(
+  private[sqs] def runSendMessageBatchRequest[R, T](failedQueue: Queue[SqsRequestEntry[T]], retryDelay: Duration, retryMaxCount: Int)(
     req: SqsRequest[T]
-  ): RIO[R with Clock, Unit] =
-    RIO.effectAsync[R with Clock, Unit]({ cb =>
-      client
-        .sendMessageBatch(req.inner)
-        .handleAsync[Unit](new BiFunction[SendMessageBatchResponse, Throwable, Unit] {
-          override def apply(res: SendMessageBatchResponse, err: Throwable): Unit =
-            err match {
-              case null =>
-                val m = req.entries.zipWithIndex.map(it => (it._2.toString, it._1)).toMap
-
-                val responsePartitioner = partitionResponse(m, retryMaxCount) _
-                val responseMapper      = mapResponse(m) _
-
-                val (successful, retryable, errors) = responseMapper.tupled(responsePartitioner(res))
-
-                val ret = for {
-                  _ <- URIO.when(retryable.nonEmpty) {
-                         failedQueue
-                           .offerAll(retryable.map(it => it.copy(retryCount = it.retryCount + 1)))
-                           .delay(retryDelay)
-                           .forkDaemon
-                       }
-                  _ <- ZIO.foreach(successful)(entry => entry.done.succeed(Right(entry.event): ErrorOrEvent[T]))
-                  _ <- ZIO.foreach(errors)(entry => entry.done.succeed(Left(entry.error): ErrorOrEvent[T]))
-                } yield ()
-
-                cb(ret)
-              case ex   =>
-                val ret = ZIO.foreach_(req.entries.map(_.done))(_.fail(ex)) *> RIO.fail(ex)
-                cb(ret)
-            }
-        })
-      ()
-    })
+  ): RIO[R with Clock with Sqs, Unit] =
+    io.github.vigoo.zioaws.sqs
```

Comment:
io.github.vigoo.zioaws.sqs
^
sorry for the stupid question, any particular reason whey `io.github.vigoo.zioaws.sqs` is used explicitly without importing it instead like:

```scala
import io.github.vigoo.zioaws.sqs._
```

and then:
```scala
...
  ): RIO[R with Clock with Sqs, Unit] =
      sendMessageBatch(req.inner)
      .mapError(_.toThrowable)
...
```

Reply:
Not at all a stupid question. I wanted to add some sort of qualifer to the method instead of just `sendMessageBatch` (where does it come from?). But `sqs.sendMessageBatch(req.inner)` doesn't compile since it looks for `zio.sqs.sendMessageBatch` namespace. Later on I used `import io.github.vigoo.zioaws` so you can do `zioaws.sqs.sendMessageBatch`